test/SourceKit/CodeComplete/complete_sequence_toplevel.swift

```diff
@@ -0,0 +1,49 @@
+class Foo {
+  var x: Int
+  var y: Int
+  func fooMethod() {}
+}
+struct Bar {
+  var a: Int
+  var b: Int
+}
+extension Bar {
+  func barMethod() {}
+}
+func foo(arg: Foo) {
+  _ = arg.
+}
+_ = Bar(a: 12, b: 42)
```

Comment:
We should have some tests that include mutation of top-level expressions (reuse) and top-level decls (no reuse).

Reply:
In this patch, for single file script, fast-completion *always* happens as long as compiler argument is not changed. It doesn't matter if it's in expressions or decls.

I will add mutation test cases anyway.